### ISSUE TITLE
Data set

### ISSUE THREAD

**Issue by vinayakumarr** [fanci-dga-detection/fanci#1]
Whether the dataset of FANCI paper is available for further research?

**Comment by sMau** [comment]
Sadly the data set is not publicly available. The source for DGA domains was DGArchive [1], which is a fee-based service but you maybe can get access for free when the data is used for research purposes only. The sources for benign NXDs are either company (Siemens AG) or university (RWTH Aachen University) owned and we are not in the position to publish this data. However, to obtain a "proper" benign data set access to any each mid to large DNS server is sufficient with subsequent filtering against DGArchive (this is also the way we obtained our benign data sets).

[1] https://dgarchive.caad.fkie.fraunhofer.de/

**Comment by vinayakumarr** [comment]
But already, deep learning algorithms are giving good results on raw Domain names. But still, you are using feature engineering with traditional machine learning. This is a resource-intensive task

**Comment by sMau** [comment]
What are you driving at?

**Comment by vinayakumarr** [comment]
I have seen around 10 papers based on deep learning based DGA detection which are already published. These are research papers only. But also I have seen one presentation video, where they implemented in real time. https://youtu.be/99hniQYB6VM?t=822

I want to know why you didn't go for deep learning algorithms. Also, you didn't cite those 10 papers in your research. My question, in most of the area already deep learning outperforming traditional machine learning algorithms. In cyber security also, it is showing the same performance. But still, you are using traditional machine learning algorithms. Could you please tell why.

**Comment by sMau** [comment]
As to limited space in the publication, related work is of course never complete. We focused on related work that compares also well in terms of their methods and not only the goal, because there is a lot of work on this topic and we cannot dicsuss each single paper. We did not want to reduce the value of any of the papers doing deep learning on this topic by not citing them. 

To answer your question why we do not use deep learning: Why should we? In our opinion, a classifier which is almost completly transparent, easy to understand, and performs well enough for real-world applicability, even on consumer-grade hardware, has more advantages than disadvantages when you are able to achieve comparable accuracies. So we decided to get the best out of "classical" ML approaches.
Nevertheless I understand your question as our approach introduces limitations espacially regarding the feature engineering as it may be used be used to introduce DGAs designed to circumvent our detection by exploiting knowledge about the features.
To conclude, we started with classical ML approaches, find them to perform very well, and then decided to continue our work relying on this approach.
I hope this helps to understand how we got to use classical ML classifiers instead of deep learning.

**Comment by vinayakumarr** [comment]
You are absolutely right. But you didn't mention in your paper by applying deep learning feature engineering step can be avoided. That's why I asked you. Very good paper and it is well written. Thank you for sharing the baseline system to the outside world for further research.
